src/main/kotlin/no/nav/familie/ef/sak/brev/VedtaksbrevService.kt

```diff
@@ -95,12 +120,42 @@ class VedtaksbrevService(private val brevClient: BrevClient,
 
         validerBeslutterIkkeErLikSaksbehandler(besluttervedtaksbrev)
 
-        val beslutterPdf = Fil(brevClient.genererBrev(besluttervedtaksbrev.tilDto(signaturMedEnhet.skjulBeslutter)))
+        val beslutterPdf = lagBeslutterPdfMedSignatur(besluttervedtaksbrev, signaturMedEnhet)
+
         val besluttervedtaksbrevMedPdf = besluttervedtaksbrev.copy(beslutterPdf = beslutterPdf)
         brevRepository.update(besluttervedtaksbrevMedPdf)
         return beslutterPdf.bytes
     }
 
+    private fun validerBehandlingKanBesluttes(behandling: Behandling) {
+        if (behandling.steg != StegType.BESLUTTE_VEDTAK || behandling.status != BehandlingStatus.FATTER_VEDTAK) {
```

Comment:
feilhvis?

Reply:
`brukerFeilHvis` - det er "nok" at saksbehandler får denne feilen, vi trenger ikke logge den som error.

Reply:
Eneste bruksmønster som kan føre til denne er at to besluttere ser på samme sak. En kommer først. Vet ikke om det er mulig å komme i denne situasjonen, men en feilmelding til bruker burde være nok. Dersom dette ikke ordner seg får vi tenke at de sier ifra til oss - de pleier jo være flinke til det :)